l10n_nl_tax_statement/views/l10n_nl_vat_statement.xml

```diff
@@ -21,7 +21,7 @@
                     <group>
                         <group name="tax_report">
                             <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
-                            <field name="date_range_id" domain="[('company_id','=',company_id)]" attrs="{'readonly': [('state','=','posted')]}"/>
```

Comment:
I believe company_id is not required on date ranges, so you might want to allow for ranges without a company here.

Reply:
Yes, true. That's why I'm removing the domain in this commit 576473fdffae884d6902ffafc4cf788975a96f5c

Reply:
LOL, sorry!